grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -1743,6 +1744,38 @@ def section_hummock(panels):
                 ),
             ],
         ),
+
+        panels.timeseries_bytes(
+            "Materialized View Read Size",
+            "",
+            [
+                
+                *quantile(
+                    lambda quantile, legend: panels.target(
+                        f'sum(histogram_quantile(0.9, sum(rate({metric("state_store_iter_size_bucket")}[$__rate_interval])) by (le, job, instance, table_id)) * on(table_id) group_left(materialized_view_id) (group({metric("table_info")}) by (materialized_view_id, table_id))) by (materialized_view_id, table_name) + sum((histogram_quantile(0.9, sum(rate({metric("state_store_get_key_size_bucket")}[$__rate_interval])) by (le, job, instance, table_id)) + histogram_quantile(0.9, sum(rate({metric("state_store_get_value_size_bucket")}[$__rate_interval])) by (le, job, instance, table_id))) * on(table_id) group_left(materialized_view_id) (group({metric("table_info")}) by (materialized_view_id, table_id))) by (materialized_view_id, table_name)',
```

Comment:
all `histogram_quantile(0.9, ...)` should be replaced by `histogram_quantile{{quantile}, ...)`

Reply:
nits: we can shorten the query by sum all histogram first and do one group_left at the end

Reply:
>sum all histogram first and do one group_left at the end

doesn't seem to work because for one internal table, its records whether iter or get are mutually exclusive. Here are three basic metrics, `iter_size`, `get_key_size` and `get_value_size`, we can only 
`iter_size group_left XXX + (get_key_size+get_value_size) group_left XXX`, but not `(iter_size+get_key_size+get_value_size) group_left XXX`